Makefile

```diff
@@ -1,32 +1,54 @@
 .PHONY: *
 BUILD_DIR=build
-COOKIECUTTER_OUTPUT_DIR=$(BUILD_DIR)/a_generated_python_project
+COOKIECUTTER_OUTPUT_DIR=$(BUILD_DIR)/a-generated-python-project
 
 help:
 	@echo "help"
 
 setup-cicd:
 	pip install --upgrade pip
-	pip install cookiecutter
+	pip install -r requirements.txt
```

Comment:
Why not use poetry for `requirements.txt` and `test_requirements.txt` ?

Reply:
Done